pkg/cli/admin/inspect/pod.go

```
@@ -283,12 +286,16 @@ func (o *InspectOptions) gatherContainerLogs(destDir string, pod *corev1.Pod, co
 	go func() {
 		defer wg.Done()
 
+		sec := (int64(o.since.Round(time.Second).Seconds()))
+		t, _ := util.ParseRFC3339(o.sinceTime, metav1.Now)
```

Ditto here `if len(o.sinceTime) > 0` and do not swallow errors!